src/components/CategoryPicker.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+export default function CategoryPicker(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="md">
+      <div className={classes.paper}>
+        <Typography component="h2" variant="h6">
+          Search with a Relationship
+        </Typography>
+        <Typography>
+          Hold shift while clicking to select more than one category to include{' '}
+        </Typography>
+        <form className={classes.form} noValidate>
```

Comment:
Is putting the form here the right thing to do? Or should you instead expect the code using this component to wrap a form around it? I don't see a submit button or anything that would cause the form to submit, so I'm not sure what purpose the form has? It seems it might also cause confusion if this component is embedded in another form.